src/main/scala/authentikat/jwt/JsonWebSignature.scala

```diff
@@ -52,7 +52,7 @@ object JsonWebSignature {
 
   abstract class Algorithm
 
-  case object none extends Algorithm
+  case object NONE extends Algorithm
```

Comment:
Ah right I'm seeing why it was like (all lower) now. None is in the default package so that's extra confusing. 
Okay NONE it is.